docker/README.md

```diff
@@ -1,54 +1,78 @@
-# Aerogear Jenkins openshift docker image
+# AeroGear Digger Jenkins OpenShift Docker images
 
-## Why another image?
+This directory contains the Docker source for Jenkins master and slaves for AeroGear Digger.
 
-OpenShift Jenkins docker images use `source to image` aproach for customizing jenkins plugins and configuration.
-We make use of the 'source to image' so that any changes made to the jenkins configuration and plugins would be persisted
+Please note that the images published to Docker Hub: <https://hub.docker.com/r/aerogear/>
 
-For base image source code and documentation please refer to https://github.com/openshift/jenkins
+From a user's point of view, you don't need to do anything here in order to build apps using AeroGear Digger.
 
-## Building
+Unless you need to change the Jenkins Docker images, you don't need to do any of the things required here.
 
-S2I build is required in order to install new plugins and configuration 
-Install s2i build tool from: https://github.com/openshift/source-to-image/releases/tag/v1.1.3 
-Execute build command:
+
+## AeroGear Digger Jenkins Master OpenShift Docker image
+
+This is the source for image in Docker Hub: <https://hub.docker.com/r/aerogear/jenkins-1-centos7/>
+
+Unless you need to change the Jenkins Docker image, you don't need to do any of the things required here.
+
+From a user's point of view, you don't need to do anything here in order to build apps using AeroGear Digger.
```

Comment:
Users would not see this. This is for devs I think. Writing here some enduser guide is not good idea. First of all we need to document this for ourselves.